test/functional/Valhalla/playlist.xml

```diff
@@ -96,6 +96,10 @@
 	</test>
 	<test>
 		<testCaseName>ValueTypeUnsafeTests</testCaseName>
+		<variations>
+			<variation>-Xcompressedrefs</variation>
+			<variation>-Xnocompressedrefs</variation>
```

Comment:
I see `ValueTypeUnsafeTests` are running with `-XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening`. We may want to add 2 variations (compressedrefs and non-compressedrefs) with flattening disabled, i.e. without `-XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening`. Some existing tests may need to be updated to take into account whether flattening is enabled.

Reply:
okay, I'll start working on all of your comments now

Reply:
addressed in 780435c